spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/TestSparkTypeConverter.java

```diff
@@ -59,6 +110,98 @@ void init() {
     gravitinoToSparkTypeMapper.put(DateType.get(), DataTypes.DateType);
     gravitinoToSparkTypeMapper.put(TimestampType.withTimeZone(), DataTypes.TimestampType);
     gravitinoToSparkTypeMapper.put(TimestampType.withoutTimeZone(), DataTypes.TimestampNTZType);
+    gravitinoToSparkTypeMapper.put(
+        ListType.of(IntegerType.get(), true), DataTypes.createArrayType(DataTypes.IntegerType));
+    gravitinoToSparkTypeMapper.put(
+        MapType.of(IntegerType.get(), StringType.get(), true),
+        DataTypes.createMapType(DataTypes.IntegerType, DataTypes.StringType));
```

Comment:
could you put struct in `gravitinoToSparkTypeMapper` too? then no need to do extral check for struct

Reply:
done